view/frontend/templates/jsstub.phtml

```diff
@@ -56,5 +58,10 @@
         window[name] = window[name] || function (cb) {
                 (window[name].q = window[name].q || []).push(cb);
             };
+        <?php if ($this->isRecoAutoloadDisabled()): ?>
+        window[name](function(api){
+            api.setAutoLoad(false);
+        });
     })();
+    <?php endif; ?>
```

Comment:
It seems this condition creates a bug, if `$this->isRecoAutoloadDisabled()` is false, the closure is never closed and creates a syntax error.